### PR TITLE
Feature/js async functions sleep

### PR DESCRIPTION
**Pull request by F-Kirchhoff**
- add a simple starter challenge for awaiting a promise

rendered: https://codesandbox.io/s/github/neuefische/web-exercises/tree/feature/js-async-functions_sleep/sessions/js-async-functions/sleep?file=/README.md

### CODE REVIEW

File: sessions/js-async-functions/sleep/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Sleep
+
+In this task we want to handle a sleep function, which can delay certain actions.
+
+## Task
+
+We have 3 buttons which should display a sleep emoji for 3, 5 and 10 seconds. After the time has passed, the Emoji should be replaced by the alarm clock. But all buttons display the alarm clock immediately.
```
Comment (djfarly) [inline sessions/js-async-functions/sleep/README.md:7]:
```suggestion
We have 3 buttons which should display a sleep emoji for 3, 5 and 10 seconds. After the time has passed, the emoji should be replaced by the alarm clock. But all buttons display the alarm clock immediately.
```

File: sessions/js-async-functions/sleep/js/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const icon = document.querySelector('[data-js="icon"]');
+const button3 = document.querySelector('[data-js="sleep-3-button"]');
+const button5 = document.querySelector('[data-js="sleep-5-button"]');
+const button10 = document.querySelector('[data-js="sleep-10-button"]');
+const body = document.body;
```
Comment (djfarly) [inline sessions/js-async-functions/sleep/js/index.js:5]:
```suggestion
```

> 'body' is assigned a value but never used. (no-unused-vars)eslint

**Review by djfarly — APPROVED** [review]
some minor things 🐈

**Comment by F-Kirchhoff** [comment]
Thanks @djfarly I added a comment and implemented your suggestions.

**Pull request merged**